IMS.BL/Application/Program.cs

```diff
@@ -25,10 +27,12 @@ public static void Main(string[] args)
             }           
             
             // Required objects.
-            var sqlConnection = SqlConnectionProvider.Instance.SqlConnectionObject;
-            var sqlInventoryRepository = new SqlInventoryRepository(sqlConnection);
+            var mongoClient = MongoConnectionProvider.Instance.MongoClient;
+            var productsCollection = new ProductCollection(mongoClient).GetProductCollection();
+            var mongoInventoryRepository = new MongoInventoryRepository(productsCollection);
+            
             var inventoryRepository = new InventoryRepository();
-            inventoryRepository.SetInventoryRepository(sqlInventoryRepository);
+            inventoryRepository.SetInventoryRepository(mongoInventoryRepository);
```

Comment:
you can combine the tow Mongo and SQL PR into the same source code and make for this configuration a factory with 2 methods one for setting things up for Mongo and one for SQL